stacks/api/function/queue/node/http.ts

```diff
@@ -87,6 +87,10 @@ export class Request {
       this.params = new Map(req.params.map(h => [h.key, h.value]));
     }
 
+    if (req.query) {
+      this.query = JSON.parse(req.query);
+    }
```

Comment:
I think we should cover this `JSON.parse()` method with try-catch block. Otherwise, the backend would down when parse progress failed just like other JSON.parse lines caused.

Reply:
Unlike other errors, this one would appear in the logs. If we catch it with a try, how would we handle it? I assume we would re-throw this error as is.